### PR TITLE
I added my name to the README

### PR DESCRIPTION
**Pull request by Najoseph**
I edited the README to add my name. It was not too challenging thanks to the step-by-step video provided.

### CODE REVIEW

**Comment by rsasschicago** [comment]
Hi Natalia! Nice job on the homework. I liked your use of the link function, and your addition of an emoji!

I only spotted two issues in your code. One is that the link to the image doesn't seem to work, and the other is that the ## is displaying as text rather than as a header. I'm not sure why that would happen. Maybe a space is required between the ## and the text?

Sincerely,

Rachel

File: README.md [hunk 1]
```diff
@@ -1 +1,19 @@
 # hw01
+
+##Background Information:
```
Comment (ktownsend3) [inline README.md:3]: Hi, Natalia! Your bio was very interesting, and it was great to learn more about you. I noticed that your header did not format correctly; perhaps, try adding a space between ## and Background Information next time.

File: README.md [hunk 2]
```diff
@@ -1 +1,19 @@
...
+My name is **_Natalia Joseph_**
+![Photo of me](https://images.app.goo.gl/iYvQDpmX9QDD6icQ6)
```
Comment (ktownsend3) [inline README.md:5]: Also, the image you attached here did not show up when I clicked the link. Maybe there was an error with the url.

File: README.md [hunk 3]
```diff
@@ -1 +1,19 @@
...
+I am currently enrolled in the *Committee on International Relations* program.
```
Comment (ktownsend3) [inline README.md:6]: Good job using the italics function!

File: README.md [hunk 4]
```diff
@@ -1 +1,19 @@
 # hw01
+
+##Background Information:
+My name is **_Natalia Joseph_**
```
Comment (ktownsend3) [inline README.md:4]: Good job using the bold function!

File: README.md [hunk 5]
```diff
@@ -1 +1,19 @@
...
+![Photo of me](https://images.app.goo.gl/iYvQDpmX9QDD6icQ6)
+I am currently enrolled in the *Committee on International Relations* program.
+I recently moved to Chicago from Mohyliv-Podilskyi, Ukraine, where I taught English. If you want more information about Mohyliv-Podilskyi, I suggest you visit the [link to the Mohyliv-Podilskyi Wikipedia page!](https://en.wikipedia.org/wiki/Mohyliv-Podilskyi)
```
Comment (ktownsend3) [inline README.md:7]: This link was embedded perfectly, and you formatted it into the biography really well!

File: README.md [hunk 6]
```diff
@@ -1 +1,19 @@
...
+
+### Past times
+
+* Exploring new places ( cities, parks, museums, countries )
+* Practicing/learning new languages
+* Cooking
+* Hanging out with friends ( when we were able to do that :sobs: )
+* Spending time with family
+
+That is about it for me. Hopefully I did this assignment correctly :joy:
```
Comment (ktownsend3) [inline README.md:19]: I did not know how to add an emoji, so thank you for teaching me that!

File: README.md (hunk 2 above)
Comment (bensoltoff) [reply README.md:5]: I think you meant

```
![](https://media-exp1.licdn.com/dms/image/C4E03AQE6MRrhNkc6Aw/profile-displayphoto-shrink_200_200/0?e=1607558400&v=beta&t=EBXwCHFifLwz0YwD823-sJQ0x_gIcwGfsz-AtMQ78ZE)
```

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |      x        |            |
| **Coding strategy**         |           |    x         |            |
| **Presentation**            |           |              |      x      |
| **Achievement, creativity** |           |       x       |            |
| **Reproducibility**         |           |       x       |            |

* Be sure to check the final version and how it looks on GitHub (or by clicking the preview button in RStudio). There are some formatting errors with your current README
* If you want text to be separated by paragraph breaks, you need to include a blank line between the different elements. Otherwise Markdown will try to render them all as a single paragraph

**Pull request closed without merging**